### PR TITLE
Update DeepConvLSTM.py - First Batchnorm is ignored

### PR DESCRIPTION
**Pull request by Nikitaz1410**
Issue: If self.use_batch_norm is True, you apply BatchNorm to inputs, but then immediately reshape the original inputs (not the batch-normed output). This means the batch-normed output is ignored.
Fix: You should reshape x, not inputs, after batch norm.

### CODE REVIEW

**Comment by ahoelzemann** [comment]
Thanks, Nikita! Much appreciated.
